test/__snapshots__/index.js.snap

```diff
@@ -81,6 +81,47 @@ export default class {
 }"
 `;
 
+exports[`works with complex spread props (class) 1`] = `
```

Comment:
Can you revert the changes to this file? It seems that you removed the fixture and the test from `test/index.js` but the snapshot is still here in the diff :)

Reply:
I added back the tests in `test/index.js`, is that what you where looking for? If not I am not sure I understand what you are asking, sorry.

Reply:
Sorry I didn't remember correctly the tests for this feature are in `test/attribute.js`. 
Can you do the following:

* remove the test from `test/index.js`
* regenerate the snapshot for `test/index.js` like so:

```
rm test/__snapshots__/index.js.snap
npm test -- -u
```